Update Frontstage docs (#5040)
iTwin/itwinjs-core: docs/learning/ui/appui-react/Frontstages.md

|**Nested** | is accessed from a primary frontstage. It may use all zones and panels, but instead of the App button, the Tool Widget contains a Back button to return to the primary frontstage.
|**Modal** | is accessed from another frontstage or the Backstage. It may contain any content along with a Back button. It does not use zones or stage panels. It is useful for application settings and data management user interfaces.

## Example Frontstage definition for displaying a viewport

### Implementing ContentGroupProvider

[ContentGroupProvider]($appui-react) describes the contents that the frontstage will contain.

```tsx
[[include:Example_Viewport_Frontstage_Group_Provider_1]]
[[include:Example_Viewport_Frontstage_Group_Provider_2]]
```

### Registering frontstage

> Stage contents are then wrapped by a [StandardFrontstageProvider]($appui-react) class and registered by [ConfigurableUiManager]($appui-react) for further use.

Reviewer: Reword "wrapped"?
```suggestion
Stage content is then provided to a [StandardFrontstageProvider]($appui-react) which is registered by [ConfigurableUiManager]($appui-react) for further use.
```

```tsx
[[include:Example_Register_Viewport_Frontstage]]
```

> Other UI items (like toolbars) contained within the frontstage should be linked with the frontstage using [UiItemsManager]($appui-abstract).

Reviewer: Reword "linked"?

```suggestion
Other UI items, like toolbar items or widgets can be provided to the frontstage using [UiItemsManager]($appui-abstract).
```
